src/Elastic.Apm/Api/User.cs

```diff
@@ -17,6 +17,7 @@ public class User
 		public string UserName { get; set; }
 
 		public override string ToString() =>
-			new ToStringBuilder(nameof(User)) { { "Id", Id }, { "UserName", UserName }, { "Email", Email } }.ToString();
+			new ToStringBuilder(nameof(User)) { { "Id",
+					Id }, { "UserName", UserName }, { "Email", Email } }.ToString();
```

Comment:
Looks like just a whitespace change - do we want this?

Reply:
Probably not. I'll remove it